src/styles/blocks/product.scss

```diff
@@ -0,0 +1,81 @@
+.product {
+  box-sizing: border-box;
+  width: $product-card-size;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 1;
+  flex-grow: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-bottom: 40px;
+    align-self: center;
+  }
+
+  &__name {
+    @include text-style(12px, 500, 18px);
```

Comment:
ok
but better to have predefined combinations

`@include text-h1`
`@include text-main`
`@include text-secondary`
`@extend %text-secondary`